src/App.js

```diff
@@ -44,6 +24,20 @@ function App() {
   const [favoriteArray, setFavoriteArray] = useState(storedFavorites);
   const navigate = useNavigate();
 
+  async function fetchGuestList() {
+    try {
+      const response = await fetch("/api/users/all");
+      const data = await response.json();
+
+      if (response.ok) {
+        const guestList = data.map((user) => user.guestList);
+        setGuestArray([...guestArray, ...guestList[0]]);
```

Comment:
I'm potentially too dazed to really tell, but I'm not sure why you are spreading the `guestList` on index 0 here. Maybe you can check for yourself whether or not that actually makes sense.

Reply:
It was a long day - I'm pretty dazed, too... 
I checked it with console.logs until it worked and that's the result 😅 
Makes sense for now, but I'll probably change it during one of the next steps...